Group-03/Tests/Surface.cpp

```
@@ -0,0 +1,40 @@
+#include "../../third-party/Catch/single_include/catch2/catch.hpp"
+
+#include <vector>
+#include "../src/Sector.h"
+#include "../src/Surface.h"
+#include "../src/Circle.h"
+
+namespace cse {
+    TEST_CASE("SectorTest 1", "Add") {
+        Sector sector(0, 0);
+        Circle circle(0, 0, 1);
+        sector.add_circle(&circle);
+        CHECK(sector.circles.size() == 1);
+    }
+
+    TEST_CASE("SectorTest 2", "Remove") {
+        Sector sector(0, 0);
+        Circle circle(0, 0, 1);
+        sector.add_circle(&circle);
+        sector.remove_circle(&circle);
+        CHECK(sector.circles.size() == 0);
+    }
+
+    TEST_CASE("SurfaceTest 1", "Add") {
```

Seems fine, you may also want to check what happens when a circle is added which is out of bounds (should probably throw an exception in debug mode at least).